src/shapes/Object/AnimatableObject.ts

```diff
@@ -53,7 +53,9 @@ export abstract class AnimatableObject<
     options: Partial<AnimationOptions<T>> = {}
   ): TAnimation<T> {
     const path = key.split('.');
-    const propIsColor = this.colorProperties.includes(path[path.length - 1]);
+    const propIsColor = AnimatableObject.colorProperties.includes(
```

Comment:
This one is odd to me but probably because other OOP languages wouldn't really support referencing a subclass like this. Is the idea that we keep AnimatableObject as is but can have a subclass with different props? Sounds good, just not something I'm used to.

Reply:
This is the way to access the static side of the class so indeed it allows you to use the static interface without limiting subclassing

Reply:
Makes sense. Just not something other languages would do. Applied as asked, although with typeof.